src/views/forms/AccountManagementForms/DepositForm/DepositButtonAndReceipt.tsx

```diff
@@ -173,6 +143,24 @@ export const DepositButtonAndReceipt = ({
           <Output type={OutputType.Asset} />
         ),
     },
+    typeof summary?.gasFee === 'number' && {
+      key: 'gas-fees',
+      label: (
+        <WithTooltip tooltip="gas-fees-deposit" stringParams={{ SOURCE_CHAIN: sourceChainName }}>
+          {stringGetter({ key: STRING_KEYS.GAS_FEE })}
+        </WithTooltip>
+      ),
+      value: <Output type={OutputType.Fiat} value={summary?.gasFee} />,
+    },
+    typeof summary?.bridgeFee === 'number' && {
+      key: 'bridge-fees',
+      label: (
+        <WithTooltip tooltip="bridge-fees">
```

Comment:
need to change to `bridge-fees-deposit` once https://github.com/dydxprotocol/v4-localization/pull/422 is merged and we should be good